README.md

```diff
@@ -2,36 +2,152 @@
 
 # devctl
 
-Command line tool for the daily development business at Giant Swarm.
+`devctl` is a command-line tool designed to streamline development workflows at Giant Swarm. It provides various commands to help manage repositories, generate files, and bootstrap applications.
 
-## Quick start
+## Installation
 
-### Installation
+```bash
+go install github.com/giantswarm/devctl/v7@latest
```

Comment:
For some reason the previous installation guide mentioned **not** to use the `go install ...` method.

Not sure what was the reason. @marians ?

Reply:
I guess it is because you need to generate some templates. The makefile handles that. I'll update the readme.